EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/76598
+webbrowsertools.com#?#div[class*="card shadow"]:contains(Advertisement)
```

Comment:
`*` is not requied. `^` is enough

Reply:
Sure.

Reply:
@Alex-302 Btw, do they even update the Popup Blocker Userscript?

Reply:
@wout-codes 
> Btw, do they even update the Popup Blocker Userscript?

What do you mean?

Reply:
@Alex-302 The AdGuard's Popup Blocker UserScript.

Reply:
About popup blocker better write to https://github.com/AdguardTeam/PopupBlocker
too many notifications in this repository, I can miss reply